src/main/java/org/apache/mesos/hdfs/HDFSScheduler.java

```diff
@@ -236,16 +251,14 @@ public void run() {
       }
     } catch (InterruptedException | ExecutionException | InvalidProtocolBufferException e) {
       log.error("Error recovering framework id", e);
-      throw new RuntimeException(e);
     }
 
-    MesosSchedulerDriver driver = new MesosSchedulerDriver(this, frameworkInfo.build(),
-        conf.getMesosMasterUri());
+    MesosSchedulerDriver driver = new MesosSchedulerDriver(this, frameworkInfo.build(), conf.getMesosMasterUri());
     driver.run();
   }
 
   private void launchNode(SchedulerDriver driver, Offer offer,
-        String nodeName, List<String> taskNames, String executorName) {
+                          String nodeName, List<String> taskNames, String executorName) {
```

Comment:
That seems to be what the formatter does too.   I would like to talk about this.   4 spaces on a line continuation makes sense... expect when the line is a method definition with parameters.  It is much easier to read params stacked IMO.

```
private void launchNode(SchedulerDriver driver, Offer offer,
    String nodeName, List<String> taskNames, String executorName) {
```
THIS is hard

```
private void launchNode(SchedulerDriver driver, 
                        Offer offer,
                        String nodeName, 
                        List<String> taskNames, 
                        String executorName) {
```
this is easier

I'll stick to the standard until we have agreement :)